package.json

```diff
@@ -24,10 +24,10 @@
   "dependencies": {
     "@babel/runtime": "^7.5.5",
     "@esteemapp/esteem-render-helpers": "^1.2.2",
-    "@ptomasroos/react-native-multi-slider": "^1.0.0",
-    "@react-native-community/cameraroll": "^1.3.0",
     "@esteemapp/react-native-autocomplete-input": "^4.2.1",
     "@esteemapp/react-native-multi-slider": "^1.1.0",
+    "@ptomasroos/react-native-multi-slider": "^1.0.0",
```

Comment:
This is not necessary, please remove this package, we have `@esteemapp/react-native-multi-slider` that replaces this.

Reply:
on recent PR merge, I have removed it, so if you remove on this branch, it won't conflict

Reply:
okay i will remove it